android/java/org/chromium/chrome/browser/ntp_background_images/util/NTPUtil.java

```diff
@@ -66,13 +79,23 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.lang.ref.SoftReference;
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Set;
 
 public class NTPUtil {
     private static final int BOTTOM_TOOLBAR_HEIGHT = 56;
     private static final String REMOVED_SITES = "removed_sites";
+    private static DisplayAd currentDisplayAd;
```

Comment:
static field naming

Reply:
replaced, as well as uses https://github.com/brave/brave-core/blob/45ef2a3938e4aba8485b6e92d1e5c523fc021552/android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java#L1188
and
https://github.com/brave/brave-core/blob/45ef2a3938e4aba8485b6e92d1e5c523fc021552/android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java#L236